components/tidb_query/src/rpn_expr/impl_json.rs

```diff
@@ -55,6 +56,83 @@ fn json_unquote(arg: &Option<Json>) -> Result<Option<Bytes>> {
     })
 }
 
+fn json_with_bytes_validator(expr: &tipb::Expr) -> Result<()> {
+    let children = expr.get_children();
```

Comment:
adding `assert!(children.len() >= 2)` is better than the comment `min_args = 2`.

Reply:
> adding `assert!(children.len() >= 2)` is better than the comment `min_args = 2`.

We already has `min_args = 2` in `rpn_fn`, so `min_args` must be no less than 2.

Reply:
"We already has min_args = 2 in rpn_fn, so min_args must be no less than 2."

That's exactly what assert means.

Reply:
> "We already has min_args = 2 in rpn_fn, so min_args must be no less than 2."
> 
> That's exactly what assert means.

I got it. It's fixed now.